ngraph/core/include/ngraph/function.hpp

```diff
@@ -138,6 +139,25 @@ namespace ngraph
         bool evaluate(const HostTensorVector& output_tensors,
                       const HostTensorVector& input_tensors) const;
 
+        /// Return a list of function's side outputs
+        const SinkVector& get_sinks(void) const { return m_sinks; }
+
+        /// \brief Add new sink nodes to the list to avoid removing
+        /// \param sinks new sink nodes
+        void add_sinks(const SinkVector& sinks);
+
+        /// \brief Delete sink node from the list of sinks
+        /// \param sink Sink to delete
+        void delete_sink(const std::shared_ptr<op::Sink>& sink);
```

Comment:
Why do we add add/delete methods only for outputs?

Reply:
Because for low latency transformation we need list of Assigns only and possiblity to add elements there.